### ISSUE TITLE
Feature Request: Clear Processing Pattern Encoder and add Item Stacks 

### ISSUE THREAD

**Issue by lilrichy** [raoulvdberge/refinedstorage#204]
It would be nice to have a "X" box on the processing pattern encoder to clear both the crafting grids like there is in the pattern grid. It is kind of annoying to have to click every item into it then back out of it to make a different pattern.

It would also be nice to be able to store item stacks in the grid for patterns that need multiple of the same item. Like many of the recipes for ender io alloys.

**Comment by raoulvdberge** [comment]
Your first paragraph I understand, but
> It would also be nice to be able to store item stacks in the grid for patterns that need multiple of the same item. Like many of the recipes for ender io alloys.
I really don't get.

Can you rephrase?

**Comment by lilrichy** [comment]
Like ender io has a recipe for fused quartz wich is 4 quartz = 1 fused quartz 
![image](https://cloud.githubusercontent.com/assets/8129775/16903665/6215c5d2-4c52-11e6-922f-70758e0bd52c.png)

Right now in the processing pattern encoder you have to do this to make it work:
![image](https://cloud.githubusercontent.com/assets/8129775/16903677/7d1057a8-4c52-11e6-9961-a00c0bad86d6.png)

it would be better if it worked like this:
![image](https://cloud.githubusercontent.com/assets/8129775/16903681/8aefd024-4c52-11e6-883b-8a08f323d161.png)

**Comment by raoulvdberge** [comment]
No, that would allow for processing patterns that take more than >9 items at a time and I don't want that edge case to be handled.

**Comment by Bartz24** [comment]
Having stacks for processing would be really nice for large recipes (especially one's for my mod Sky Resources)
![combustion](https://cloud.githubusercontent.com/assets/18143350/17087272/53dbaad6-51d4-11e6-8dfb-1ecd7c32532b.PNG)
As you can see, some (about 20+ recipes) need more than 9 items.

**Comment by raoulvdberge** [comment]
Alright, I'll try to implement this for the next version
